src/nvector/mpiplusx/fmod/fnvector_mpiplusx_mod.f90

```diff
@@ -152,20 +152,20 @@ function FN_VGetArrayPointer_MPIPlusX(v) &
 use, intrinsic :: ISO_C_BINDING
 real(C_DOUBLE), dimension(:), pointer :: swig_result
 type(N_Vector), target, intent(inout) :: v
-type(C_PTR) :: fresult 
-type(C_PTR) :: farg1 
+type(C_PTR) :: fresult
+type(C_PTR) :: farg1
 
 farg1 = c_loc(v)
 fresult = swigc_FN_VGetArrayPointer_MPIPlusX(farg1)
-call c_f_pointer(fresult, swig_result, [1])
+call c_f_pointer(fresult, swig_result, [FN_VGetLocalLength_MPIPlusX(v)])
```

Comment:
`N_VGetLocalLength_MPIPlusX` is missing from the `.h` and `.c` files

Reply:
It is there https://github.com/LLNL/sundials/blob/main/include/nvector/nvector_mpiplusx.h#L47. It just is not in the diff because it already existed.

Reply:
Yep, I was looking for it in the diff but it's already there. All good.